### PR TITLE
dark mode!

### PR DESCRIPTION
**Pull request by gcpetri**

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -16,50 +16,27 @@ ActiveStorage.start()
...
-const themeDarkBlue = "#3A546C";
-const themeBackgroundColor = "#F0F0F0";
-const whiteColor = "#FFF";
-
-function requestEventPassword(eventId, eventName) {
-    console.log(eventId);
-    console.log(eventName);
-}
-
-function addShortcutKeys() {
```
Comment (SyraAM) [inline app/javascript/packs/application.js:29]: Was this function ever used / was it deleted here intentionally?

Comment (gcpetri) [reply app/javascript/packs/application.js:29]: i don’t think it was ever used. it only ever had console logs in it anyway

File: app/models/user.rb [hunk 2]
```diff
@@ -39,6 +39,21 @@ def count_attended
     events.size
   end
 
+  def self.count_points(user_id)
+    current_event_types = EventTypes.all
+    point_hash = {}
+    current_event_types.each do |event_type|
+      point_hash[event_type.id] = event_type.pointValue
+    end
+    this_users_events = Event.joins(:events_users)
```
Comment (SyraAM) [inline app/models/user.rb:48]: This is already implemented as `user.events` and `user.events_users`

**Review by SyraAM — CHANGES_REQUESTED** [review]
The admin login and admin points queries pages need styling; both have black text on a black background in dark mode.

**Comment by gcpetri** [comment]
> The admin login and admin points queries pages need styling; both have black text on a black background in dark mode.

People try to use scss variables instead of magic colors

File: app/models/user.rb (hunk 2 above)
Comment (gcpetri) [reply app/models/user.rb:48]: Does it though? I didn't see those using event types

**Comment by SyraAM** [comment]
The sun and moon switcher icons are fun but too prominent on the page; I would suggest switching to the Material Icons [Light Mode](https://fonts.google.com/icons?selected=Material%20Icons%3Alight_mode%3A) and [Dark Mode](https://fonts.google.com/icons?selected=Material%20Icons%3Adark_mode%3A) symbols, at a smaller size.

File: app/models/user.rb (hunk 2 above)
Comment (SyraAM) [reply app/models/user.rb:48]: `this_users_events` is; I'm making a branch `point-filtering` to add filtering to this function real quick so I can refactor it as well

File: app/javascript/packs/application.js (hunk 1 above)
Comment (SyraAM) [reply app/javascript/packs/application.js:29]: All good then haha

**Review by SyraAM — APPROVED** [review]

**Review by bensp97 — APPROVED** [review]
Looks Good!

Code passes all relevant tests (RSpec, Rubocop, Brakeman, etc.)

Meets relevant acceptance criteria and definition of done.

**Pull request merged**